src/hotspot/share/opto/superword.cpp

```diff
@@ -749,86 +742,6 @@ int SuperWord::get_iv_adjustment(MemNode* mem_ref) {
   return iv_adjustment;
 }
 
-//---------------------------dependence_graph---------------------------
-// Construct dependency graph.
-// Add dependence edges to load/store nodes for memory dependence
-//    A.out()->DependNode.in(1) and DependNode.out()->B.prec(x)
-void SuperWord::dependence_graph() {
```

Comment:
Note: new code is in `VLoopDependencyGraph::construct`. The general idea is the same, but I refactored it a bit.